README.md

```diff
@@ -0,0 +1,22 @@
+# WTExtractor
+Tools to work with WildTanget game assets
+## Pywtools
```

Comment:
I think we should have a 2nd readme in pywttools that describes how to install and use it.